### DISCUSSION TITLE
[iOS] Banner notification while app is in the foreground?

### DISCUSSION THREAD

**Discussion by GuidoNeele** [shinyorg/shiny#1055] [Q&A] [answered]
https://github.com/shinyorg/shiny/blob/e5d0cbc177f97ec5037d31299065cb1d7b740923/src/Shiny.Core/Platforms/Apple/ShinyUserNotificationDelegate.cs#L52-L68

Since UNNotificationPresentationOptions.Alert is deprecated maybe Banner is a better default for showing notifications while the app is in the foreground?
https://developer.apple.com/documentation/usernotifications/unnotificationpresentationoptions

**Comment by aritchie** [comment, marked as answer]
I will definitely look into this.  Thanks for the heads up.  It's tough to keep track of all the platform changes going on right now

**Reply by aritchie** [reply]
This will be part of 2.6 and is already in 3.0
